README.md

```diff
@@ -77,3 +85,84 @@ For homebrew, see the following: https://gist.github.com/mxcl/1173223
 If you take these steps, obviously also remove your ROS workspace and start the install
 process over from scratch as well. Finally, audit your `$PATH` variable to ensure that
 when you run `python`, you're getting Homebrew's `python`.
+Another way to check which Python you are running is to do:
+
+```bash
+which python # Should result in /usr/local/bin/python
+ls -l $(which python) # Should show a symlink pointing to Homebrew's Cellar
+```
+
+If you are getting permission errors when you `sudo uninstall` pip packages,
+see [Issue #11](https://github.com/mikepurvis/ros-install-osx/issues/11) and
+[this StackOverflow Q&A](http://stackoverflow.com/a/35051066/2653356).
+
+### El Capitan support
+
+The `install` script may not work as smoothly in OS X El Capitan.
+Here are some pointers, tips, and hacks to help you complete the installation.
+This list was compiled based on the discussion in [Issue #12](https://github.com/mikepurvis/ros-install-osx/issues/12).
+
+#### No definition of [procps] for OS [osx]
+
+This issue is known to the developers and is being addressed. See:
+
+* http://answers.ros.org/question/224956/no-definition-of-procps-for-os-osx
+* https://github.com/ros-perception/vision_opencv/pull/109
+
+#### FLTK fails to build
+
+For a quick fix, you can do the following:
+
+```bash
+brew install fltk --devel
+
+```
+
+#### library not found for -ltbb
+
+See [Issue #4](https://github.com/mikepurvis/ros-install-osx/issues/4).
+You need to compile using Xcode's Command Line Tools:
+
+```shell
+xcode-select --install # Install the Command Line Tools
+sudo xcode-select -s /Library/Developer/CommandLineTools # Switch to using them
+gcc --version # Verify that you are compiling using Command Line Tools
+```
+
+The last command should output something that includes the following:
+
+```bash
+Configured with: --prefix=/Library/Developer/CommandLineTools/usr
+```
+
+You'll then have to rerun the entire `install` script or do the following:
+
+```bash
+rm -rf /opt/ros/indigo/* # More generally, /opt/ros/${ROS_DISTRO}/*
+rm -rf build/ devel/ # Assuming your working dir is the catkin workspace
+catkin build \
+  ... # See actual script for the 4-line-long command
+```
+
+#### dyld: Library not loaded
+
+This may occur after installation has finished successfully and you try to
+execute something like `rosrun`. For example:
+
+```bash
+rosrun turtlesim turtlesim_node
+dyld: Library not loaded: librospack.dylib
+  Referenced from: # Some file in the catkin ws
+  Reason: image not found
+find: ftsopen: No such file or directory
+[rosrun] Couldnt find executable named turtlesim_node below
+find: ftsopen: No such file or directory
+```
+
+The quick fix is to add
+
+```bash
+export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:/opt/ros/indigo/lib
```

Comment:
It should be addressed upstream. The problem is probably that `rosrun` calls `rospack` in a sub-shell and the library path is not inherited because of SIP. We could either ensure that a proper RPATH is set between `rospack` (or any other executable/library) and `librospack.dylib`. This would make it so that we do not depend on the `DYLD_LIBRARY_PATH` being set. Otherwise, we might be able to modify `rosrun` so that it doesn't need a sub-shell or that it runs the sub shell in such a way as to preserve the dyld library path.